app/src/main/java/de/danoeh/antennapod/adapter/StatisticsListAdapter.java

```diff
@@ -0,0 +1,105 @@
+package de.danoeh.antennapod.adapter;
+
+import android.content.Context;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.BaseAdapter;
+import android.widget.ImageView;
+import android.widget.TextView;
+
+import com.bumptech.glide.Glide;
+import com.joanzapata.iconify.widget.IconTextView;
+
+import java.util.ArrayList;
+import java.util.Locale;
+
+import de.danoeh.antennapod.R;
+import de.danoeh.antennapod.core.feed.Feed;
+import de.danoeh.antennapod.core.glide.ApGlideSettings;
+import de.danoeh.antennapod.core.storage.DBReader;
+
+/**
+ * Adapter for the statistics list
+ */
+public class StatisticsListAdapter extends BaseAdapter {
+    private Context context;
+    ArrayList<DBReader.StatisticsItem> feedTime = new ArrayList<>();
+
+    public StatisticsListAdapter(Context context) {
+        this.context = context;
+    }
+
+
+    @Override
+    public int getCount() {
+        return feedTime.size();
+    }
+
+    @Override
+    public DBReader.StatisticsItem getItem(int position) {
+        return feedTime.get(position);
+    }
+
+    @Override
+    public long getItemId(int position) {
+        return 0;
+    }
+
+    @Override
+    public View getView(int position, View convertView, ViewGroup parent) {
+        FeedHolder holder;
+        Feed feed = feedTime.get(position).feed;
+
+        if (convertView == null) {
+            holder = new FeedHolder();
+            LayoutInflater inflater = (LayoutInflater) context
+                    .getSystemService(Context.LAYOUT_INFLATER_SERVICE);
+
+            convertView = inflater.inflate(R.layout.nav_feedlistitem, parent, false);
+
+            holder.image = (ImageView) convertView.findViewById(R.id.imgvCover);
+            holder.title = (TextView) convertView.findViewById(R.id.txtvTitle);
+            holder.failure = (IconTextView) convertView.findViewById(R.id.itxtvFailure);
+            holder.count = (TextView) convertView.findViewById(R.id.txtvCount);
+            convertView.setTag(holder);
+        } else {
+            holder = (FeedHolder) convertView.getTag();
+        }
+
+        Glide.with(context)
+                .load(feed.getImageUri())
+                .placeholder(R.color.light_gray)
+                .error(R.color.light_gray)
+                .diskCacheStrategy(ApGlideSettings.AP_DISK_CACHE_STRATEGY)
+                .fitCenter()
+                .dontAnimate()
+                .into(holder.image);
+
+        holder.title.setText(feed.getTitle());
+        holder.failure.setVisibility(View.GONE);
+        holder.count.setVisibility(View.VISIBLE);
+        holder.count.setText(timeToText(feedTime.get(position).timePlayed));
+        return convertView;
+    }
+
+    private String timeToText(long time) {
+        int hours = (int) time / 3600;
```

Comment:
I prefer a conversion that is done automatically: The method could also return days if the number of hours gets too high without requiring additional code at other places